share/spack/gitlab/cloud_e4s_pipelines/stacks/e4s/spack.yaml

```diff
@@ -123,6 +123,7 @@ spack:
       - spack --version
       - cd share/spack/gitlab/cloud_e4s_pipelines/stacks/e4s
       - spack env activate --without-view .
+      # - spack config add "config:install_tree:projections:${SPACK_JOB_SPEC_PKG_NAME}:'morepadding/{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'"
```

Comment:
@chuckatkins This is how we intend to test binary relocation.  I commented it out in the WIP commit to see if it was the problem, and indeed it seemed to be what broke `automake`, so I still need to dig into that.  But you could try adding this line to your `spack.yaml` in #22179 to see what happens.